README.md

```diff
@@ -1,15 +1,19 @@
 # @pulumi/eslint-plugin
+
```

Comment:
These spacing changes are from `npm run format`. It actually also adds changes to some `.github/workflow/*.yml` files. I'm ignoring them in this PR cuz they are too unrelated.